.nvmrc

```diff
@@ -0,0 +1 @@
+lts/boron
```

Comment:
Is this intentional?

Reply:
Yes; this is the version specified in the readme. When I tried npm installing with the latest node, there was no appropriate fs-events version. I'm not sure what the real fix for that would be, but I figured the nvmrc and readme should agree with each other anyway.